money_test.go

```diff
@@ -23,6 +23,14 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestCurrency(t *testing.T) {
```

Comment:
Don't relay on currencies we have now it can be changed after time.
My suggestion is if you want to test Currency please create one (We can say `mock`) with `Currency.AddCurrency()` and make tests with it.